kyolo/utils/bounding_box_utils.py

```diff
@@ -0,0 +1,92 @@
+from typing import List, Tuple
+
+from keras import KerasTensor, Model, ops
+
+
+def generate_anchors(image_size: List[int], strides: List[int]):
+    """
+    Find the anchor maps for each w, h.
+
+    Args:
+        image_size List: the image size of augmented image size
+        strides List[8, 16, 32, ...]: the stride size for each predicted layer
+
+    Returns:
+        all_anchors [HW x 2]:
+        all_scalers [HW]: The index of the best targets for each anchors
+    """
+    img_w, img_h = image_size
+    anchors = []
+    scaler = []
+    for stride in strides:
+        anchor_num = img_w // stride * img_h // stride
+        scaler.append(ops.full((anchor_num,), stride))
+        shift = stride // 2
+        h = ops.arange(0, img_h, stride) + shift
+        w = ops.arange(0, img_w, stride) + shift
+        anchor_h, anchor_w = ops.meshgrid(h, w, indexing="ij")
+        anchor = ops.stack(
+            [ops.reshape(anchor_w, -1), ops.reshape(anchor_h, -1)], axis=-1
+        )
+        anchors.append(anchor)
+    all_anchors = ops.cast(ops.concatenate(anchors, axis=0), dtype="float32")
```

Comment:
Do we need 32 bit precision here? if not, backend.floatx() could be better to use?